packages/sn-filter-pane/src/hooks/listbox/get-listbox-resources.ts

```diff
@@ -1,8 +1,15 @@
 import { IFilterPaneLayout, ListboxResourcesArr } from '../types';
 
-async function getListboxResourcesFromIds(app: EngineAPI.IApp, ids: string[]): Promise<ListboxResourcesArr> {
+async function getListboxResourcesFromIds(app: EngineAPI.IApp, ids: string[], layout: IFilterPaneLayout): Promise<ListboxResourcesArr> {
```

Comment:
Easier to read if distinguish list box layout from filter pane layout:
```suggestion
async function getListboxResourcesFromIds(app: EngineAPI.IApp, ids: string[], fpLayout: IFilterPaneLayout): Promise<ListboxResourcesArr> {
```